modules/swagger-codegen/src/main/resources/rust-server/client.mustache

```diff
@@ -269,8 +269,8 @@ impl Api for Client {
                     let response_{{name}} = response.headers.get::<Response{{nameInCamelCase}}>().ok_or_else(|| "Required response header {{baseName}} for response {{code}} was not found.")?;
 {{/headers}}
 
-{{#dataType}}                    Ok({{operationId}}Response::{{message}}{{^headers}}(body){{/headers}}{{#headers}}{{#-first}}{ body: body, {{/-first}}{{name}}: response_{{name}}.0.clone(){{^-last}}, {{/-last}}{{#-last}} }{{/-last}}{{/headers}})
-{{/dataType}}{{^dataType}}                    Ok({{operationId}}Response::{{message}}{{#headers}}{{#-first}}{ {{/-first}}{{^-first}}, {{/-first}}{{name}}: response_{{name}}.0.clone(){{#-last}} }{{/-last}}{{/headers}})
+{{#dataType}}                    Ok({{operationId}}Response::{{#vendorExtensions}}{{x-responseId}}{{/vendorExtensions}}{{^headers}}(body){{/headers}}{{#headers}}{{#-first}}{ body: body, {{/-first}}{{name}}: response_{{name}}.0.clone(){{^-last}}, {{/-last}}{{#-last}} }{{/-last}}{{/headers}})
+{{/dataType}}{{^dataType}}                    Ok({{operationId}}Response::Status{{code}}{{#headers}}{{#-first}}{ {{/-first}}{{^-first}}, {{/-first}}{{name}}: response_{{name}}.0.clone(){{#-last}} }{{/-last}}{{/headers}})
```

Comment:
I suspect this might be the source of the errant `Statusxxx` I've seen further down. Should this be `{{#vendorExtensions}}{{x-responseId}}{{/vendorExtensions}}` instead of `Status{{code}}`?

Reply:
I was careless, thanks for pointing out.